glean-core/src/glean.udl

```diff
@@ -35,6 +35,7 @@ namespace glean {
     boolean glean_set_debug_view_tag(string tag);
     boolean glean_set_source_tags(sequence<string> tags);
     void glean_set_log_pings(boolean value);
+    void glean_set_log_level(string log_level);
```

Comment:
This should take an `enum LogLevel` instead of a stringy API.
See https://mozilla.github.io/uniffi-rs/udl/enumerations.html